crisp/__main__.py

```diff
@@ -296,6 +296,12 @@ def do_main(args, cfg):
         return
     w.accept(n_code, ('main', 'transpile'))
 
+    n_code = w.split_ffi(n_code)
+    if not w.test(n_code, n_c_code):
+        print('error: tests failed after split_ffi')
```

Comment:
nit: Printing errors to stderr can be more useful.

Reply:
Leaving this as is for now.  We have a lot of prints in different places and I think it's best to have it all going to the same place, so that if stdout and stderr are redirected to different files, there's less risk of confusion about the ordering of messages.